### PR TITLE
Add an optional log to the `now_playing` segment

### PR DESCRIPTION
**Pull request by IngoMeyer441**

### CODE REVIEW

**Comment by xx4h** [comment]
Hi, thanks for contributing! Do you mind fixing the shell_shfmt and bash linter errors?

**Comment by IngoMeyer441** [comment]
@xx4h Thanks for your response. Should all be ok now :smiley:.

**Review by xx4h — CHANGES_REQUESTED** [review]

File: segments/now_playing.sh [hunk 1]
```diff
@@ -120,6 +126,33 @@ run_segment() {
 		return ${exitcode}
 	fi
 	if [ -n "$np" ]; then
+		if [[ -n "$TMUX_POWERLINE_SEG_NOW_PLAYING_TRACK_LOG_FILEPATH" && -n "$TMUX_POWERLINE_SEG_NOW_PLAYING_TRACK_LOG_MAX_ENTRIES" ]]; then
```
[the remark this answers is not in the document]
Comment (IngoMeyer441) [reply segments/now_playing.sh]: I think this is a good idea. I have added the bool flag and a small helper function to a new lib file to check the flag value (I hope this is ok).

[the remark this answers is not in the document]
Comment (xx4h) [reply segments/now_playing.sh]: > I think this is a good idea. I have added the bool flag and a small helper function to a new lib file to check the flag value (I hope this is ok).

yeah, that's great, i like the `lib/util.sh`.

**Review by xx4h — APPROVED** [review]

**Pull request merged**